tests/features/camel-core/src/test/java/org/apache/karaf/camel/itest/CamelCoreITest.java

```diff
@@ -38,7 +35,7 @@ protected String getTestBundleName() {
 
     @Override
     protected List<String> getRequiredFeatures() {
-        return List.of();
+        return List.of("camel-blueprint");
```

Comment:
A bit hacky so we need to add a comment to explain why

Reply:
Done